db/seeds.rb

```diff
@@ -186,34 +186,18 @@
 
 ## TIME SLOTS / APPOINTMENTS ##
 
-ubsf_america.create_time_slot_generation_config!(ubs_id: ubsf_america.id).tap do |config|
-  config[:windows] = [
-    { start_time: '07:40', end_time: '08:00', slots: 6 },
-    { start_time: '08:00', end_time: '16:20', slots: 8 },
-    { start_time: '16:20', end_time: '22:00', slots: 4 }
-  ]
-  config[:max_appointment_time_ahead] = 0
-  config.save!
-end
-
-create_slot = lambda do |attributes|
-  Appointment.create!(attributes)
-end
-
-generation_service = TimeSlotGenerationService.new(create_slot: create_slot)
-
-worker = TimeSlotGenerationWorker.new(
-  time_slot_generation_service: generation_service
-)
-
-worker_opts = TimeSlotGenerationWorker::Options.new(
-  sleep_interval: 5.seconds,
-  execution_hour: current_time.hour
-)
-
-# mimic successful worker.execute
-worker.generate_ubs_time_slots(ubsf_america, worker_opts, current_time)
-TimeSlotGeneratorExecution.where(date: current_time.to_date).update_all(status: 'done')
+TimeSlotGenerationService.new(create_slot: lambda { |attrs| Appointment.create(attrs) })
```

Comment:
Using new TimeSlotGenerationService as it's more reliable when creating appointments.